src/views/evaluation/Evaluation.js

```diff
@@ -0,0 +1,122 @@
+const NAV = function (option = {}){
+    return `
+    <ul class="header headerT">
+        ${logoButton(option.Logo)}
+        ${constructNav(option)}
+    </ul>`
+}
+
+const logoButton = function(option = {}){
+    return `
+    <li class="header-elements">
+        <img class= "main-logo" src="${option}">
+    </li>`
+}
+
+const buttonNav = function(option={}, active="" , specialPlacement = ""){
```

Comment:
Answered at the question underneath , and I put by mistake option will look into it soon and remake the naming scheme to make it more readable